### ISSUE TITLE
Fable incorrectly mangles static member name

### ISSUE THREAD

**Issue by WildOgr** [fable-compiler/Fable#564]
### Description

Fable incorrectly mangles static member name

### Repro steps

Sample code
```
#r "../node_modules/fable-core/Fable.Core.dll"
#load "../node_modules/fable-import-vscode/Fable.Import.VSCode.fs"
 
open Fable.Core
open Fable.Core.JsInterop
open Fable.Import
open Fable.Import.vscode

let activate (context : vscode.ExtensionContext) = 
  commands.registerCommand("extension.sayHello", fun _ ->
    window.showErrorMessage "Err" |> unbox )
  |> context.subscriptions.Add  
```

### Expected behavior

Expected to produce
```
_vscode.window.showErrorMessage("Err");
```

for ``` window.showErrorMessage "Err" ```
### Actual behavior
Produces ```_vscode.window.showErrorMessage_0("Err");```
Full output:

```
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.activate = activate;

var _vscode = require("vscode");

function activate(context) {
  (function () {
    var objectArg = context.subscriptions;
    return function (arg00) {
      objectArg.push(arg00);
    };
  })()(_vscode.commands.registerCommand("extension.sayHello", function (_arg1) {
    return _vscode.window.showErrorMessage_0("Err");
  }));
}
```
### Related information 

* Fable Version 0.7.9
* .NET Runtime 4.6
* Operating system Windows

**Comment by alfonsogarciacaro** [comment]
Thanks for the report, this should be fixed in fable-compiler 0.7.10

**Comment by Dessix** [comment]
I've confirmed that it works here, except in one circumstance I will be documenting as a bug once I can get a minimal reproduction prepared.
